plugins/package-json-hook/.toolkitrc.yml

```diff
@@ -1,4 +1,5 @@
 installs:
-  PackageJson: './lib/package-json-helper'
+  PackageJson:
+    entryPoint: './lib/package-json-helper'
```

Comment:
**suggestion(non-blocking)**: should we add the managed files as part of this PR?

```suggestion
    entryPoint: './lib/package-json-helper'
    managedFiles:
      - 'package.json'
```